Cargo.toml

```diff
@@ -56,74 +57,89 @@ primitives = { path = "primitives", default-features = false }
-frame-executive = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-frame-election-provider-support = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-frame-support = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-frame-system = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-frame-system-benchmarking = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-frame-system-rpc-runtime-api = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-pallet-balances = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-pallet-insecure-randomness-collective-flip = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-pallet-session = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-pallet-staking-substrate = { package = "pallet-staking", branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-pallet-scheduler = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-pallet-sudo = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-pallet-timestamp = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-pallet-transaction-payment = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-pallet-transaction-payment-rpc = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git" }
-pallet-transaction-payment-rpc-runtime-api = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", default-features = false }
-sc-basic-authorship = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git" }
-sc-chain-spec = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git" }
-sc-cli = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git" }
-sc-client-api = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git" }
-sc-client-db = { branch = "polkadot-v0.9.40", git = "https://github.com/paritytech/substrate.git", features = [
+frame-benchmarking-cli = { branch = "pos-keep-history", git = "https://github.com/gluwa/substrate.git" }
+frame-benchmarking = { branch = "pos-keep-history", git = "https://github.com/gluwa/substrate.git", default-features = false }
```

Comment:
I'd like to enable as many of the upstream CI test jobs against our fork as possible.  So that we can have some sort of confidence when upgrading this branch. 

@nathanwhit can you enable GitHub actions executions on the forked repository ? 

I will take a look at their test jobs in the mean time.

Reply:
I see some GHA notifications on the branch so CI jobs must be running. I will open another ticket for myself to dig into this further. It wouldn't be as easy as I though initially.